ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!--
+PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
+-->
+
+## I'm submitting a...
+<pre><code>
+[ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
+[ ] Feature request
+[ ] Documentation issue or request
+</code></pre>
+
+## Current behavior
+<!-- Describe how the issue manifests. -->
+
+
+## Expected behavior
+
+## Minimal reproduction of the problem with instructions
+
+## If this is a feature request please fill out the following:
+<pre><code>
+As a (user, developer, contributor, etc):
+I want:
+So that:
+<!-- Describe the motivation or the concrete use case. -->
+</code></pre>
+
+## Environment
+<pre><code>
+Browser:
```

Comment:
Since this is a Gtk program, I don't think that the web browser information is needed.